src/SFA.DAS.PSRService.Web/Controllers/ReportController.cs

```diff
@@ -78,20 +80,119 @@ public IActionResult Create()
         [Route("Create")]
         [Authorize(Policy = PolicyNames.CanEditReport)]
         public IActionResult PostCreate()
+        {
+            return RedirectToAction("IsLocalAuthority", "Report");
+        }
+
+        [HttpGet]
+        [Route("IsLocalAuthority/{id}")]
+        [Route("IsLocalAuthority")]
+        [Authorize(Policy = PolicyNames.CanEditReport)]
+        public IActionResult IsLocalAuthority(bool? id)
+        {
+            var report = _reportService.GetReport(_currentPeriod.PeriodString, EmployerAccount.AccountId);
+
+            var isLocalAuthorityViewModelVm = new IsLocalAuthorityViewModel();
+
+            if (report != null)
+            {
+                if (!_reportService.CanBeEdited(report))
+                    return new RedirectResult(Url.Action("Index", "Home"));
+                isLocalAuthorityViewModelVm.IsLocalAuthority = id.HasValue ? id : report.IsLocalAuthority;
+                ViewBag.CurrentPeriod = report.Period ?? _currentPeriod;
+            }
+            else
+            {
+                ViewBag.CurrentPeriod = _currentPeriod;
+            }
+
+            return View("IsLocalAuthority", isLocalAuthorityViewModelVm);
+        }
+
+        [HttpPost]
+        [Route("PostIsLocalAuthority")]
```

Comment:
I don't think this has changed, we shouldn't have the httpmethod as the prefix to the route name